src/tasks/MultiFormPurgeTask.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace SilverStripe\MultiForm\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\MultiForm\Models\MultiFormSession;
+use SilverStripe\ORM\DataList;
+use SilverStripe\ORM\DataObject;
+
+/**
+ * Task to clean out all {@link MultiFormSession} objects from the database.
+ *
+ * Setup Instructions:
+ * You need to create an automated task for your system (cronjobs on unix)
+ * which triggers the process() method through cli-script.php:
+ * `php framework/cli-script.php MultiFormPurgeTask`
+ * or
+ * `framework/sake MultiFormPurgeTask`
+ *
+ * @package multiform
+ */
+class MultiFormPurgeTask extends BuildTask
+{
+    /**
+     * Days after which sessions expire and
+     * are automatically deleted.
+     *
+     * @var int
+     */
+    private static $session_expiry_days = 7;
+
+    /**
+     * Run this cron task.
+     *
+     * Go through all MultiFormSession records that
+     * are older than the days specified in $session_expiry_days
+     * and delete them.
+     */
+    public function run($request)
+    {
+        $sessions = $this->getExpiredSessions();
+        $delCount = 0;
+        if ($sessions) {
+            foreach ($sessions as $session) {
+                $session->delete();
+                $delCount++;
+            }
+        }
+        echo $delCount . ' session records deleted that were older than ' . $this->config()->get('session_expiry_days') . ' days.';
```

Comment:
I know we don't provide a better way than direct output, but maybe we should at least add `PHP_EOL` to the end so it won't just stack up on one line =)